src/parsers/perf/perfparser.cpp

```diff
@@ -1009,6 +1012,35 @@ class PerfParserPrivate : public QObject
         }
     }
 
+    void addTracepointToTracepointStack(uint32_t tid, const Data::Tracepoint& tracepoint)
+    {
+        auto addToLookup = [this](const QString& name) -> bool {
+            for (const auto& parameters : tracepointParameters) {
+                auto match = parameters.startRegex.match(name);
```

Comment:
ok, now that I understand what you are doing here:

isn't this the same as

```
auto stopName = name.replace(parameters.startRegex, parameters.stopExpression);
if (stopname == name)
    continue; // not matched
```

Reply:
yes